### PR TITLE
ENT-5028 PULL Sync single org method API implementation

### PR DESCRIPTION
**Pull request by chaynesred**
https://issues.redhat.com/browse/ENT-5028

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in Ibutsu:
https://url.corp.redhat.com/ibutsu-runs-406c568a-734e-4f6f-aa4b-b6b43ca84d2b

File: swatch-core/src/main/resources/swatch-core/application.yaml [hunk 1]
```diff
@@ -139,6 +139,7 @@ rhsm-subscriptions:
   package_uri_mappings:
     # this mapping required here because it is used by our SecurityConfig, which is shared
     org.candlepin.subscriptions.resteasy: ${PATH_PREFIX}/${APP_NAME}/v1
+    org.candlepin.subscriptions.conduit.resteasy: ${PATH_PREFIX}/${APP_NAME}/v1
```
Comment (kahowell) [inline swatch-core/src/main/resources/swatch-core/application.yaml:142]: Is it possible to move this to swatch-system-conduit/src/main/resources/application.yaml ? I *think* it should work just as well there, if not fine to keep it here.

**Review by kahowell — CHANGES_REQUESTED** [review]
Overall, looks good. One tiny potential change I'd like to suggest (see previous comment).

File: swatch-core/src/main/resources/swatch-core/application.yaml (hunk 1 above)
Comment (chaynesred) [reply swatch-core/src/main/resources/swatch-core/application.yaml]: @kahowell got this update in, could you take a look?Thanks!

**Review by kahowell — APPROVED** [review]
LGTM, feel free to squash and merge

**Pull request merged**